### PR TITLE
Update DownloadTrait.php

### LINKED ISSUE #160: PHP Notice:  Undefined index: local in .../src/DownloadTrait.php on line 51
@blacksenator: das passiert wenn das Config File noch auf altem Stand ist. Können wir nochmal überlegen, ob die `local` Section im Config File sinnvoll ist? Wann braucht man das?

### PR DESCRIPTION
**Pull request by disc**
Fixes #160
If no local config found, convert to empty array

### CODE REVIEW

File: src/DownloadTrait.php [hunk 1]
```diff
@@ -48,7 +48,11 @@ public function downloadAllProviders(OutputInterface $output, bool $downloadImag
     {
         $vcards = [];
 
-        foreach ($this->config['local'] as $file) {
+        $localConfig = $this->config['local'] ?? [];
+        if (!$localConfig) {
+            error_log('Notice: Local config is empty and will not be applied');
```
Comment (andig) [inline src/DownloadTrait.php]: Why do we need the error message?

**Comment by andig** [comment]
Wouldn‘t it be more obvious to just have a command line parameter for local files?

File: src/DownloadTrait.php (hunk 1 above)
Comment (disc) [reply src/DownloadTrait.php]: I think this message will useful for cases when no local config passed, because there is no error anymore.

**Comment by disc** [comment]
> Wouldn‘t it be more obvious to just have a command line parameter for local files?

My point is: there should be a validation of config property. Local config key should be exists in any case.

**Comment by andig** [comment]
See #161 for a different approach.

**Pull request closed without merging**